Statistics.cs

```diff
@@ -64,29 +70,87 @@ public void CompareCitiesPopulation()
 
         }
 
-        public void ShowCityOnMap()
+        /// <summary>
+        /// Show a location on the google maps site
+        /// </summary>
+        /// <param name="lat">Lattitude of the location</param>
+        /// <param name="lng">Longitude of the location</param>
+        public void ShowCityOnMap(double lat, double lng)
         {
-
+            string url = $"https://www.google.com/maps/@{lat},{lng},15z";
+            try
+            {
+                Process.Start(new ProcessStartInfo("cmd", $"/c start {url}") { CreateNoWindow = true });
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Error launching Internet Explorer\nError: " + ex.Message);
+            }
         }
 
-        public void CalculateDistanceBetweenCities()
+        /// <summary>
+        /// Determine the distance between two points using the Bing Map API
+        /// </summary>
+        /// <param name="lat1">Lattitude of the first city</param>
+        /// <param name="lng1">Longitude of the first city</param>
+        /// <param name="lat2">Lattitude of the second city</param>
+        /// <param name="lng2">Longitude of the second city</param>
+        /// <returns>A awaitable task to ensure that the method is called</returns>
+        public async Task CalculateDistanceBetweenCities(double lat1, double lng1, double lat2, double lng2)
```

Comment:
I can make this work

Reply:
Yeah, when you implement it, you will have to change the main to a async Task instead of just returning void. Also have to await it when you call it.